addons/mail/static/src/models/thread/thread.js

```diff
@@ -2086,6 +2108,19 @@ function factory(dependencies) {
             default: '',
             dependencies: ['orderedOtherTypingMembers'],
         }),
+        /**
+         * Text that represent the url of the document to be used in an href attribute
```

Comment:
_"URL to access to the conversation"_ should be good enough description, and straight to the point.